src/main.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import App from './App';
+import './index.css';
+
```

Comment:
- [ ] Kindly fix the error in the image below, it is something to do with imports.

![vite](https://user-images.githubusercontent.com/53124548/220154348-a8e6806f-ef19-4f27-b8fd-9eff8cd45cff.png)